__tests__/e2e/scripts/ci_integration.sh

```diff
@@ -0,0 +1,22 @@
+SYNTHETICS_E2E_ARGS="${@:2}"
+
+#!/usr/bin/env bash
+set -e
```

Comment:
The shebang should be added at the beginning of the script, so this might cause some issues